utils/install.sh

```diff
@@ -750,6 +750,14 @@ main() {
     fi
 
     if [ $PLUGINS_INSTALL -eq 1 ]; then
+        if [ "$(printf %s\\n%s\\n "0.10.1-alpha.1" "$VERSION")" != "$(printf %s\\n%s "0.10.1-alpha.1" "$VERSION" | awk '{ if ($1 ~ /-/) print; else print $0"_" ; }' | sort --version-sort | sed 's/_$//')" ]; then
```

Comment:
Is this limited to `Linux`? Currently, our WASI-NN with OpenVINO support is only for linux(ubuntu 20.04).

Reply:
Yes please see https://github.com/WasmEdge/WasmEdge/commit/2d750c79ac7d3f3a69cab6e24e8b165b2351c930#diff-001b8193af60259ffc2829010784899f35764c864992fe3e96a606ad2650bb2bR631
Line 631